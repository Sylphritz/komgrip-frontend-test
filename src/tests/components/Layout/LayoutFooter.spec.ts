import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import LayoutFooter from '@/components/Layout/LayoutFooter.vue'

describe('LayoutFooter', () => {
  it('renders successfully and matches the snapshot', () => {
    const wrapper = mount(LayoutFooter, {
      shallow: true
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
