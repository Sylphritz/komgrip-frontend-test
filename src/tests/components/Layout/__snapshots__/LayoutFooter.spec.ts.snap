// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LayoutFooter > renders successfully and matches the snapshot 1`] = `
<footer
  class="footer pt-3"
>
  <div
    class="container-fluid"
  >
    <div
      class="row align-items-center justify-content-lg-between"
    >
      <div
        class="col-lg-12 mb-lg-0 mb-4"
      >
        <div
          class="copyright text-center text-sm text-muted text-lg-start"
        >
           Copyright Â© 2022 Komgrip Technologies Co., Ltd. 
        </div>
      </div>
    </div>
  </div>
</footer>
`;
